dkg-runtime-primitives/src/lib.rs

```diff
@@ -75,12 +95,30 @@ pub const DKG_ENGINE_ID: sp_runtime::ConsensusEngineId = *b"WDKG";
 // Key type for DKG keys
 pub const KEY_TYPE: sp_application_crypto::KeyTypeId = sp_application_crypto::KeyTypeId(*b"wdkg");
 
+// Max length for proposals
+pub type MaxProposalLength = CustomU32Getter<10_000>;
+
+// Max authorities
+pub type MaxAuthorities = CustomU32Getter<10_000>;
+
+// Max reporters
+pub type MaxReporters = CustomU32Getter<10_000>;
+
+/// Max size for signatures
+pub type MaxSignatureLength = CustomU32Getter<10_000>;
+
+/// Max size for signatures
+pub type MaxKeyLength = CustomU32Getter<10_000>;
+
+/// Max votes to store onchain
+pub type MaxVotes = CustomU32Getter<10_000>;
+
```

Comment:
These values needs a second thought actually, like for example, the `MaxSignatureLength` should be 64 bytes.